catalog-rest-service/src/main/java/org/openmetadata/catalog/security/DefaultAuthorizer.java

```diff
@@ -132,6 +153,7 @@ public boolean hasPermissions(AuthenticationContext ctx, EntityReference owner)
   public boolean hasPermissions(
       AuthenticationContext ctx, EntityReference entityReference, MetadataOperation operation) {
     validateAuthenticationContext(ctx);
+    // TODO: Should check if we should return true if entity does not have an owner.
```

Comment:
@harshach 

After giving this some thought, I think we may have to do this here:

1. If owner == null, return true
2. If user from authCtx == owner of entity, return true
3. do policy evaluation (which facilitates RBAC)

Reply:
+1